### PR TITLE
 on reserve and extending resources all fields updated in table, appropriate buttons disabled till view refresh. Fixes #10,  Fixes #12

### LINKED ISSUE #10: When clicking Extend on an Expired resource its status should change to "Valid"
@bergano65 
I clicked an expired resource and it remained Expired.
The Expiration date was updated, but the status wasn't. The status should also be updated, and the "Extend" button should become greyed out so that nobody thinks about clicking it again.

### LINKED ISSUE #12: Update default expiration intervals
Unclaimed : 10 days
Use 30 days for a "claimed resource"
Reserve : 180 days

### PR DESCRIPTION
**Pull request by bergano65**

### CODE REVIEW

File: CogsMinimizer/Views/Subscription/Analyze.cshtml [hunk 1]
```diff
@@ -36,9 +36,12 @@
             }),
             contentType: "application/json; charset=utf-8",
             success: function (data) {
-                // Let's update expiration date
+                // Let's update expiration date, validity status, resource owner
                 // Model at server is updated at controller method and will be used for table complete updating at refresh table
+                $selectedRow.find('td').eq(3).html(data.Owner);
                 $selectedRow.find('td').eq(5).html(data.ExpirationDate);
+                $selectedRow.find('td').eq(6).html("Valid");
```
Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: You shouldn't hard code any values.
Just use the updated status that returns from the controller

File: CogsMinimizer/Views/Subscription/Analyze.cshtml [hunk 2]
```diff
@@ -56,9 +59,12 @@
...
```
Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: same here

**Comment by shklar** [comment]
@bergano65 
I have approved and merged the PR.
Please fix the comments I wrote in a separate PR.

**Comment by bergano65** [comment]
You were right, much more convenient keep in scope of tasks problems for which issues are created. Couldn’t you make issue?

**Comment by shklar** [comment]
Make an Issue about what?
Feel free to create issues.

**Comment by bergano65** [comment]
About small issues you merging  for now but would be fixed in next submit. )

**Comment by shklar** [comment]
Didn't think it was worth a dedicated issue

**Comment by bergano65** [comment]
I noticed  your remark about removing hard coded vals from sources. Something else?s

**Pull request merged**